src/17-infinite-discovery/data.md

```diff
@@ -0,0 +1,76 @@
+# How to generate the data
+
+This documents the Gravity console snippets that were used to generate the
+datasets that go into this experiments data directory:
+
+- `17-infinite-discovery/data/artworks.json`
+
+These files are gitignored due to their size and currently live in a shared
+drive instead:
+
+- https://drive.google.com/drive/u/1/folders/1Lh7msUc0R_JlpNEzApZ4YbqB8x5tbpws
+
+The files were generated via the following Ruby snippets which were entered
+directly into a Gravity Rails console.
+
+## Choose collections
+
+For a large-ish dataset (a few thousand works), select all manually curated
+collections:
+
+```ruby
+collections = CuratedMarketingCollection.pluck(:id)
+```
+
+## Select artworks and their associated artists and partners
+
+```ruby
+# artworks
+artworks = collections.map{ |id|
+  MarketingCollection.find(id).artworks.published.for_sale
+}.flatten.uniq.compact
+```
+
+## Serialize the artworks
+
+Here we opt to use collector-facing terminology rather than Artsy internal
+jargon, on the assumption that this would perform better with the embedding
+models. For example:
+
+- attribution_class → **rarity**
+- category → **medium**
+- medium → **materials**
+- genes → **categories**
+- etc.
+
+We also keep some basic info about the associated artists and partners.
```

Comment:
```suggestion
We also keep some basic info about the associated artists.
```

Looks like this dataset drops the associated partner data, which makes sense. Just a stray copypasta here…

Reply:
And L25 above as well